nototools/tool_utils.py

```diff
@@ -303,3 +332,38 @@ def emit():
     start = prev = v
   emit()
   return sep.join(num_list)
+
+
+def setup_logging(loglevel, quiet_ttx=True):
+  """Set up logging to stream to stdout.
+
+  The loglevel is a logging level name or a level value (int or string).
+
+  ttx/fontTools uses 'info' to report when it is reading/writing tables,
+  but when we want 'info' in our own tools we usually don't want this detail.
+  When quiet_ttx is true, set up logging to treat 'info' logs from
+  fontTools misc.xmlReader and ttLib as though they were at level 19."""
```

Comment:
@dougfelt you don't need a `Filter` for this. If you simply want to mute, say, `info()` messages from one logger, you can simply set that specific logger's `level` to a value that is higher than the root logger's level.
In fonttools, we follow the convention to have one logger instance per module, and name it after the module itself.

```python
if loglevel == logging.INFO:
    for module in ('fontTools.misc.xmlReader', 'fontTools.ttLib'):
        logger = logging.getLogger(module)
        logger.setLevel(loglevel + 1)
```

I agree that ttLib and xmlReader are a bit noisy (they were like that already, so I kept them as is). But at least now that we use `logging` clients can mute them more easily.